crates/router/src/connector/wise.rs

```diff
@@ -113,6 +114,8 @@ impl ConnectorCommon for Wise {
     }
 }
 
+impl ConnectorValidation for Wise {}
```

Comment:
default implementation is not called from build_request method

Reply:
fn build_request is not implemented for wise

Reply:
Will remove ` impl ConnectorValidation for Wise`